docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`-` Correctly handle a remote error in the Aerodrome/Velodrome pool querying logic.
```

Comment:
Please don't make me repeat myself every time. Messages in the changelog should be user facing. What is a remote error, what is a pool querying logic? These are implementation details the user does not need to know.

```suggestion
* :bug:`-` A rare error with velodrome events decoding should now be fixed and the relevants events properly decoded.
```